opus/application/apps/ui/views.py

```diff
@@ -283,6 +284,14 @@ def _update_form_with_grouping(form, form_vals, glabel, gvalue):
     auto_id = True
     selections = {}
     extras = {}
+    customized_input = slug in settings.MULT_WIDGETS_WITH_TOOLTIPS
```

Comment:
We shouldn't need MULT_WIDGETS_WITH_TOOLTIPS anymore. Instead look through the mult values and see if any of them have tooltips. If they do, then you need a customized input. If they're all None, then you don't need one.